pkgs/by-name/in/invidtui/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, yt-dlp, ffmpeg, mpv }:
+
+buildGoModule rec {
+  pname = "invidtui";
+  version = "0.3.6";
+
+  src = fetchFromGitHub {
+    owner = "darkhz";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-zUr0zrIJPpqhHvL7PFFN7cgcgBXV+WHO/eRes7+HzxM=";
+  };
+
+  vendorHash = "sha256-cKvY3/3N3SESBVol7Af3M3mJaPwxLzd/rKN8P+qh7sY=";
+
+  doCheck = true;
+
+  postPatch = ''
+        substituteInPlace cmd/flags.go \
+          --replace "\"ffmpeg\"" "\"${ffmpeg}/bin/ffmpeg\"" \
+          --replace "\"mpv\"" "\"${mpv}/bin/mpv\"" \
+          --replace "\"yt-dlp\"" "\"${yt-dlp}/bin/yt-dlp\""
+  '';
+
+  meta = with lib; {
+    homepage = "https://darkhz.github.io/";
+    description =
+      "invidtui is an invidious client, which fetches data from invidious instances and displays a user interface in the terminal(TUI), and allows for selecting and playing Youtube audio and video.";
```

```suggestion
    description = "An invidious TUI client";
```

Descriptions should be short and to the point.